order/src/main/java/com/yas/order/model/Order.java

```diff
@@ -1,18 +1,40 @@
 package com.yas.order.model;
 
 
+import com.yas.order.model.enumeration.EDeliveryMethod;
+import com.yas.order.model.enumeration.EDeliveryStatus;
+import com.yas.order.model.enumeration.EPaymentMethod;
 import jakarta.persistence.*;
 import lombok.*;
 
+import java.math.BigDecimal;
+import java.time.ZonedDateTime;
+import java.util.Set;
+
 @Entity
 @Table(name = "order")
 @Getter
 @Setter
 @NoArgsConstructor
 @AllArgsConstructor
 @Builder
-public class Order {
+public class Order extends AbstractAuditEntity{
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
+    private String phone;
+    private String address;
+    private String note;
+    private float tax;
+    private float discount;
+    private int numberItem;
+    private BigDecimal totalPrice;
+    private BigDecimal deliveryFee;
+    private EDeliveryMethod deliveryMethod;
+    private EDeliveryStatus deliveryStatus;
+    private EPaymentMethod paymentMethod;
+
+    @OneToMany(mappedBy = "orderId", fetch = FetchType.EAGER, cascade = CascadeType.ALL)
+    Set<OrderDetail> orderDetails;
```

Comment:
Consider initialize value in this line like Set<OrderDetail> orderDetails = new HashSet().

Reply:
Order always Order Item inside, thus, i dont think we need to initialize OrderItem(OrderDetail) when we initialize Order. we will set it on function "createOrder " in the later PR